utils/cliutils/commandsflags.go

```diff
@@ -997,6 +998,11 @@ var flagsMap = map[string]cli.Flag{
 		Name:  passphrase,
 		Usage: "[Optional] The passphrase for the signing key. ` `",
 	},
+	distTarget: cli.StringFlag{
+		Name: target,
+		Usage: "[Optional] The target path for artifacts matched in the pattern path. " +
```

Comment:
The target path for artifacts matched in the pattern path. --> The target path for distributed artifacts on the edge node. If not specified, the artifacts will have the same path and name on the edge node, as on the source Artifactory server.